src/components/textarea-autosize/TextareaAutosize.js

```diff
@@ -75,6 +76,7 @@ export default class TextareaAutosize extends Component {
         // Create new comments when pressing enter without shift
         if (event.key === 'Enter' && !event.shiftKey) {
             event.preventDefault();
+            findDOMNode(this).blur();
```

Comment:
Why was the blur added here? We might want to blur for replies but not for new comments.

Reply:
This event is fired for new comments and new replies. Are you saying that we should only `blur` it when we are replying, instead of `blur` it in both cases?

Reply:
Yep, when I'm adding a new comment on the bottom bar, the textarea should be kept focused.

Reply:
I've removed this line.